index.md

```diff
@@ -1,5 +1,14 @@
 svrx(server-x) is a platform built for efficient front-end development.
 
+## Feature List
+
+- **create static server** in current dir
+- easy to **proxy** everything
+- **auto reload**: editing css will update styles only, other files will reload the whole page
```

Comment:
Support [HMR](https://webpack.js.org/concepts/hot-module-replacement/) out of box